front/src/components/exampleForRedux/ExampleForRedux.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import actions from "../../common/redux/actions/";
+import store from "../../common/redux/store";
+import { Heading, Wrapper, Ptag } from "./styledComponentsExample";
+
+class ExampleForRedux extends Component {
+    constructor(props) {
+        super(props);
+        const unsubscribe = store.subscribe(() => {
+            const check = store.getState();
+            if (check.isLoad) {
+                document.title = "React is loaded";
+                unsubscribe();
+            }
+        });
+    }
+    render() {
+        store.dispatch(actions.calculate("+", 180, 6));
```

Comment:
Please, don't import store directly, use `react-redux` for dispatching actions

Reply:
done